foreman-selinux-enable

```diff
@@ -17,20 +18,22 @@ is_redhat_6() {
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    /usr/sbin/semanage port -E > $TMP_PORTS
+
+    # Remove previously defined elasticsearch_port_t
+    # (this can be removed in future release)
+    grep elasticsearch_port_t $TMP_PORTS | sed s/-a/-d/g >> $TMP
```

Comment:
How about this - there are now two temp files and they only gets executed when non-empty. I haven't tested this but in case there is no typo this should work and be flexible for the future.